Makefile

```diff
@@ -23,7 +23,8 @@ test: generate
 # testacc runs acceptance tests
 testacc: generate
 	@if [ "$(TEST)" = "./..." ]; then \
-		echo "ERROR: Set TEST to a specific package"; \
+		echo "ERROR: Set TEST to a specific package. For example,"; \
+		echo "  make TEST=builtin/providers/aws/resource_aws_instance.go testacc"; \
```

Comment:
I actually usually put the `TEST` at the end to make it easier to edit line to line. Can we change it to that?